src/backend/distributed/commands/common.c

```diff
@@ -272,3 +274,67 @@ PreprocessDropDistributedObjectStmt(Node *node, const char *queryString,
 
 	return NodeDDLTaskList(NON_COORDINATOR_NODES, commands);
 }
+
+
+/*
+ * DropTextSearchDictObjectAddress returns object address for the drop TextSearchDict stmt.
+ */
+ObjectAddress
+DropTextSearchDictObjectAddress(Node *node, bool missing_ok)
+{
+	DropStmt *stmt = castNode(DropStmt, node);
+
+	ObjectAddress objectAddress = { 0 };
+
+	List *objName = NIL;
+
+	foreach_ptr(objName, stmt->objects)
```

Comment:
Hmm, we are iterating over the `stmt->objects` but only return one objectAddress.

Could we ever have multiple objects in `stmt->objects`? If so, what can we do? If not, can we avoid the loop?